slash/plugins/builtin/xunit.py

```diff
@@ -75,11 +75,13 @@ def session_end(self):
         if config.root.parallel.worker_id is not None:
             return
 
+        suite_time = sum([result.get_duration().total_seconds() for result in context.session.results.iter_test_results()])
```

Comment:
timedelta objects support summing -
`sum((result.get_duration() for result in results), datetime.timedelta())`

Reply:
done